File: patterns/javascript-patterns/pattern66.js
// Pattern 66: ...

// 0
// *1
// **2
// ***3
// ****4
// *****5

const n = 5;

for (let i = 0; i <= n; i++) {
    let row = '';

    for (let j = 0; j <= i; j++) {
        if (i > j) {
            row += '*';
        } else {
            row += i;
        }
    }

    console.log(row);
}

File: patterns/python-patterns/pattern66.py
# Pattern 66: ...

# 0
# *1
# **2
# ***3
# ****4
# *****5

n = 5

for i in range(n + 1):
    for j in range(i + 1):
        if i > j:
            print("*", end="")
        else:
            print(i, end="")
    print()
